test/plugins/webhooks.helper.test.js

```diff
@@ -1436,7 +1435,7 @@ describe('startHookEvent test', () => {
                         branch: Promise.resolve('master')
                     })
                 ]);
-            const pipelineMock2 = _.cloneDeep(pipelineMock);
+            const pipelineMock2 = JSON.parse(JSON.stringify(pipelineMock));
```

Comment:
Using `structuredClone()` would be cleaner than having to force everything to be JSON compatible.
```suggestion
            const pipelineMock2 = structuredClone(pipelineMock);
```

Reply:
@minghay - not sure if we need to consider older browser versions. 
<img width="791" alt="Screenshot 2024-05-06 at 3 25 20 PM" src="https://github.com/screwdriver-cd/screwdriver/assets/7771180/6b4be9be-7bcb-4bac-a7c3-2c371833ef6c">